agents/heuristics.py

```diff
@@ -0,0 +1,60 @@
+from arguments import get_arguments
```

Comment:
Please create a PR with just the heuristics file (i.e. branch of current main, add the heuristics file, and then create the PR)